docs/TabbedForm.md

```diff
@@ -881,6 +880,58 @@ const ProductEdit = () => (
 
 Check [the RBAC `<TabbedForm>` component](./AuthRBAC.md#tabbedform) documentation for more details.
 
+
+To use the `<TabbedForm>` component, you will work with the `<TabbedForm.tab>` component. Since `<TabbedForm>` is provided by the `@react-admin/ra-rbac` package, then `<TabbedForm.tab>` too. This component will only renders a tab and its content if the user has the right permissions.
```

Comment:
```suggestion
To use the `<TabbedForm>` component, you will work with the `<TabbedForm.Tab>` component. Since `<TabbedForm>` is provided by the `@react-admin/ra-rbac` package, then `<TabbedForm.Tab>` too. This component will only render a tab and its content if the user has the right permissions.
```

Be sure to use Pascal case for component names.

Also, I'm not found of the wording.

But to me there is a bigger issue: if we are not going to make separate sections for `<TabbedForm>` and `<TabbedForm.Tab>`, then it doesn't make much sense to provide 2 code snippets that are almost the same.
We should instead provide a single code snippet, using both `<TabbedForm>` and `<TabbedForm.Tab>` from `ra-rbac`, and explaining in the comments that it can:
- hide the Delete button
- hide the Stock tab 
- hide the Description and Image inputs

Reply:
But before taking actions on this one, maybe you should wait for @fzaninotto opinion.